libraries/lib-track/Track.h

```diff
@@ -305,7 +305,9 @@ class TRACK_API Track /* not final */
     @pre `IsLeader()`
     @post result: `NChannels() == result->NChannels()`
     */
-   virtual TrackListHolder Duplicate() const;
+   virtual TrackListHolder Duplicate(
+      std::optional<std::pair<double, double>> unstretchInterval = std::nullopt,
+      std::function<void(double)> reportProgress = [](double) {}) const;
```

Comment:
The default can simply be '{}' provided then that you null-check it where you use it